src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx

```diff
@@ -468,7 +468,7 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
                         dataStore={this.props.store.dataStore}
                         vizWidth={this.props.geneWidth}
                         vizHeight={130}
-                        xMax={this.props.store.pfamGeneData.result.length}
+                        xMax={this.props.store.pfamGeneData.result.length || this.props.store.gene.length / 3}
```

Comment:
order of operations stuff makes me uneasy.. can you put parens around the division